contents/blog/interview-snapshot-guide.md

```diff
@@ -0,0 +1,76 @@
+
+## Filling out your interview snapshot
+
+Once you've reviewed your notes, you're ready to start filling out your snapshot. You can use the template below, or recreate it in Google Docs, Figma etc.
+
+![interview-snapshot](../images/interview-snapshots/interview-snapshot.jpg)
+
+### 1. Note high-level information of your user
+
+To set the scene, add the user’s name at the top of the interview snapshot – add the job role and company name for B2B companies for extra for context. It’s also handy to add the date of the interview and a high-level summary what the interview was about. If available, add a picture of the user and the company logo, if the context is B2B. This is optional, but I find it helps trigger my memory when I revisit interviews a few weeks after the fact.
+
+### 2. Add any relevant background information
+
+What information about your user is relevant to your company and product? Their age? Their responsibilities in their job role? What does their morning routine looks like? Write this information down in the “quick facts” column. I also recommend writing down any previous experience the user has with your product here – i.e. are they a long-term user, or have they recently started using your product?
+
+### 3. List the opportunities you’ve identified
+
+Did the user express any pain points or desires in relation to your product or problem area? Write these down in the “Opportunities” column. During the interview, make sure you really get to the bottom of the user’s needs, and avoid the **XY problem**. Make sure to list problems here, and **not feature ideas**.
+
+> The [**XY problem**](https://xyproblem.info/) describes when you ask a user for a solution Y to their problem X. If you build Y without understanding X, you are missing important context that would allow you to build a solution that actually solves the problem.
+
+### 4. Note any other interesting insights
+
+If the user mentioned anything interesting that doesn’t fit in the “Opportunities” column, you can add it to “Insights”. This can include anything that isn’t a pain point, a need or a desire. Did the user show any uncommon behaviour? Did they share anything about why they chose your product (or not)? If there is anything you want your colleagues to know about without reading the whole interview transcript, this is the place to put it.
+
+### 5. Add one or two memorable quotes
+
+At the top of the template, there is space to add one or two memorable quotes from the interview. This is useful, for example, if the user shared a particular emotion in the interview you want to relay word by word, and you want to create empathy in your team for this particular issue.
+
+### 6. Share the interview snapshot with your team
+
+Take a screenshot of your completed interview snapshot and share it with your team – we have a dedicated Slack channel for them. Snapshots are bound to create a conversation, follow up questions or ideas. This is exactly what you want.
```

Comment:
Per Charles's feedback, I think this would be opportunity to add some kind of example of this debate in action at PostHog – e.g. a time we changed something immediately based on an interview / snapshot.